javascript-forms/form-events.js

```diff
@@ -0,0 +1,35 @@
+/* define a function named handleFocus with parameter(event);
+logs "focus" when event is fired;
+logs "event.target.name";
+document.querySelector for element with ID user-name
+create variable for that element;
+use variable for addEventListener */
+
+/* define a function named handleBlue with parameter(event);
+logs"blur" when blur event fired;
+logs "event.target.name";
+use variable for addEventListener */
+
+/* define a function named handleInput with parameter(event);
+logs "event.target.name";
+logs "event.target.value";
+use variable for addEventListener */
+
+function handleFocus(event) {
+  console.log('focus event fired:', event);
+  console.log('event.target.name:', event.target.name);
+}
+var $userName = document.querySelector('#user-name');
+$userName.addEventListener('focus', handleFocus);
```

Comment:
You need to have each of these event handlers applied to all three elements; each element will have three event listeners applied to them for `focus`, `blur`, and `input`